### PR TITLE
issue #98 - urls that are to long, break the layout

### PR DESCRIPTION
**Pull request by kotzendekrabbe**

### CODE REVIEW

File: docs/css/src/issues-list.css [hunk 1]
```diff
@@ -10,6 +10,10 @@
   border-left: 10px solid;
 }
 
+.issue-url {
+  word-break: break-word;
```
Comment (zoepage) [inline docs/css/src/issues-list.css]: that is an invalid property. did you mean `break-all`? :)

**Review by zoepage — CHANGES_REQUESTED** [review]

File: docs/css/src/issues-list.css (hunk 1 above)
Comment (kotzendekrabbe) [reply docs/css/src/issues-list.css]: upsi, yes. sorry. will fix it - thank you :)

Comment (zoepage) [reply docs/css/src/issues-list.css]: <3

**Comment by kotzendekrabbe** [comment]
@zoepage  fixed :)

**Review by zoepage — APPROVED** [review]

**Pull request merged**